ports/renesas-ra/Makefile

```diff
@@ -343,10 +369,12 @@ SRC_O += \
 SRC_O += \
 	shared/runtime/gchelper_thumb2.o
 
+ifneq ($(BOARD),$(filter $(BOARD),VK_RA4W1 VK_RA6M3 VK_RA6M5))
```

Comment:
I suggest you use a `$(wildcard $(BOARD_DIR)/*.c)` here so other boards can have `board_init.c` files.

Reply:
Every other bords **will have** `board_init.c` and `board_led.c`, except the VK ones. These files exist only if a given board has a BSP in the `FSP`, so every board not part of some kind of program or arangment with Renesas will be wiped out  with next upgrade of the FSP. It si pointless to add them to all boards unless you are sure that all boards will be included in every next new FSP release.

Reply:
Sorry didn't notice the `ifneq`, I think the exception should be the official EK boards (because every new board will Not have those files, so will have to be added to the Makefile) ,so I think the condition should check for official boards not custom boards, or better maybe just  ?

```
EK_BOARD_DIR = $(HAL_DIR)/ra/board/$(BOARD_LOW)
ifneq ($(wildcard $(TOP)/$(EK_BOARD_DIR)/.),)
HAL_SRC_C += $(addprefix $(EK_BOARD_DIR)/,\
	board_init.c \
	board_leds.c \
	)
endif
```

Reply:
Well I have power over VK boards, for the EK ask @TakeoTakahashi2020.

Reply:
@mbedNoobNinja , @iabdalkader ,  I have confirmed that the building of RA4M1_CLICKER is successful with EK_BOARD_DIR and BOARD_LOW before the code edited by @iabdalkader.  So FSP_BOARD_DIR and FSP_BOARD_NAME also shoud be OK.  I agree with this change.